db/schema.rb

```diff
@@ -73,8 +73,8 @@
 
   create_table "issue_assignments", force: :cascade do |t|
     t.integer  "issue_id"
-    t.datetime "created_at",                           null: false
-    t.datetime "updated_at",                           null: false
+    t.datetime "created_at"
+    t.datetime "updated_at"
     t.integer  "repo_subscription_id"
```

Comment:
The default database constraints are getting removed. Unless it is intentional, would suggest that you do not make any changes not related to your story in this PR. Getting it back to as it is on master. 

This issue might have caused due to some Postgres version difference.